app/controllers/any_login/application_controller.rb

```diff
@@ -11,12 +14,19 @@ class ApplicationController < ActionController::Base
     end
 
     def any_login
-      try_not_to_leak_any_login_is_installed
       head 403 && return unless AnyLogin.verify_access_proc.call(self)
       add_to_previous
       AnyLogin.provider.constantize::Controller.instance_method(:any_login_sign_in).bind(self).call
     end
 
+    def css
+      send_file AnyLogin::Engine.root.join('app', 'assets', 'stylesheets', 'any_login', 'application.css'), content_type: 'text/css'
+    end
+
+    def js
+      send_file AnyLogin::Engine.root.join('app', 'assets', 'javascripts', 'any_login', 'application.js'), content_type: 'text/javascript'
```

Comment:
This worked properly in my local testing, I believe it will work properly when built as a gem as well.